lib/travis/hub/model/log.rb

```diff
@@ -14,4 +14,12 @@ def clear
     update_column(:removed_by, nil)
     Part.where(log_id: id).delete_all
   end
+
+  def add(msg)
+    if aggregated_at
+      update_column(:content, [content, msg].join("\n"))
+    else
+      Part.create!(log_id: id, content: msg, final: true)
```

Comment:
No worries :)

I've made it so that Hub talks to Logs via HTTP now. https://github.com/travis-ci/travis-hub/pull/82/commits/83a7a3d89ef23adfd3d2cd9a3042eb37bdaea356

Interestingly GitHub doesn't do that "commented on outdated diff" thing for reviews, apparently.